### ISSUE TITLE
How to handle Cubemaps [V2]

### ISSUE THREAD

**Issue by simeonradivoev** [Unity-Technologies/PostProcessing#551]
How do we sample cubemaps using the version 2 of the post processing stack ?
I'm using Unity 2018.1 and assigned cubemaps don't show up the standard samplerCUBE. Are there plans to put macros for sampling cubemaps ?

**Comment by Chman** [comment]
Indeed, we're missing cubemap-related macros as we never needed them until now.
